src/app/components/pages/PickAccount.jsx

```diff
@@ -285,7 +284,11 @@ class PickAccount extends React.Component {
                         style={{ maxWidth: '36rem', margin: '0 auto' }}
                     >
                         <br />
-                        <Progress tabIndex="0" value={10} max={100} />
+                        <progress max="100" value="10">
```

Comment:
why use `<progress>` at all? we'll have to do custom styles for each browser anyways (only webkit styles are included in this PR); we should just do the `div.progress` i think

![screenshot from 2018-01-25 09-46-44](https://user-images.githubusercontent.com/99194/35402068-65a8ea66-01c9-11e8-94c4-f9d30c696581.png)

Reply:
That's what this is, it's html5 <progress> element, not a component...

Reply:
it'll get native styling, and is ugly though (see the win7 example)

here's chrome (looks nice)

![screenshot from 2018-01-25 12-27-02](https://user-images.githubusercontent.com/99194/35402627-353a776c-01cb-11e8-9b5d-900336bda0e8.png)


and here's ff (blue, weird)

![screenshot from 2018-01-25 12-27-12](https://user-images.githubusercontent.com/99194/35402633-399efd5a-01cb-11e8-93cc-1a775572520a.png)